fastlane/Fastfile

```diff
@@ -439,16 +489,19 @@ platform :ios do
     ensure_sentry_installed
 
     unless options[:skip_prechecks]
-      ios_build_prechecks(skip_confirm: options[:skip_confirm], external: true)
+      ios_build_prechecks(
+        skip_confirm: options[:skip_confirm],
+        external: true
+      )
       ios_build_preflight
-      xcversion() # Ensure we're using the right version of Xcode, defined in `.xcode-version` file
+      xcversion # Ensure we're using the right version of Xcode, defined in `.xcode-version` file
```

Comment:
I know `xcversion` has been deprecated in favor of `xcodes`, but I decided to hold off on switching over to `xcodes` until I could fully test out the change.